app/models/partner.rb

```diff
@@ -33,8 +33,7 @@ class Partner < ApplicationRecord
   # Validations
   validates :name,
             presence: true,
-            uniqueness: true,
-            case_sensitive: false,
```

Comment:
The rest of this file is just altering double quotes to single and changing some formatting to remove linter warnings.  This line is the one that fixes an error during tests. Essentially, `case_sensitive` has always been a specific argument of uniqueness, not a validator by itself. Rails now requires that to be explicit.